nginx_configs/albs.conf

```diff
@@ -64,4 +68,9 @@ server {
         }
         proxy_pass http://pulp;
     }
+
+    location /sign-file/ {
```

Comment:
In production I remember that we have separate locations for `login` and `sign` endpoints. I don't remember the reasoning behind it, but if it works with only 1 location then we're good